index.html

```diff
@@ -32,90 +32,89 @@
   </head>
 
   <body>
-    <div class="first-screen">
```

Comment:
Was removing this Div Really required?

Reply:
yeah, it was required as the navbar and main content container  was in one container so have to separate them.